views/blocks/profiling_tags.tpl

```diff
@@ -11,6 +11,26 @@
 
 [{if $oPayment->isCustomPaymentMethod()}]
 
+  [{if $oPayment->oxpayments__oxid->value === 'wdratepay-invoice'}]
+
+  [{assign var="sDeviceIdentToken" value=$oViewConf->getRatepayUniqueToken()}]
+  <script language="JavaScript">
```

Comment:
I guess this is just copied over, but the `language` attribute has been deprecated for a long time and should not be used.

Reply:
@tomazpu should I delete the language attribute?

Reply:
yes

Reply:
👍